### ISSUE TITLE
Looks like it only works on the CPU, even if it solves the problem "NameError: name '_C' is not defined"

### ISSUE THREAD

**Issue by ws1hope** [IDEA-Research/Grounded-Segment-Anything#274]
Both the model and the data need to be moved to the GPU.

**Comment by mhyeonsoo** [comment]
@ws1hope 
Have you set enviroments?
I'd faced the same issue, and solved by setting up the environment first before installing GroundingDino and diffusers.

```shell
export AM_I_DOCKER=False
export BUILD_WITH_CUDA=True
export CUDA_HOME=/path/to/cuda-11.3/
```

**Comment by ws1hope** [comment]
> 

yes, i have set environments, however, this demo can not run on GPU. Then, i move model and data to GPU, it can run on gpu correctlly.

**Comment by LiaoZihZrong** [comment]
rebuild it again, and it will OK, i don't know why
`python -m pip install -e GroundingDINO`
